src/algebra/algebra/basic.lean

```diff
@@ -124,6 +124,74 @@ end prio
 def algebra_map (R : Type u) (A : Type v) [comm_semiring R] [semiring A] [algebra R A] : R →+* A :=
 algebra.to_ring_hom
 
+namespace algebra_map
+
+def has_lift_t (R A : Type*) [comm_semiring R] [semiring A] [algebra R A] :
+  has_lift_t R A := ⟨λ r, algebra_map R A r⟩
```

Comment:
Probably we should have a simp lemma that either unfolds or refolds this spelling. In the long term, refolding is probably the right choice, but that `simp_nf` linter might have hundreds of complaints to make about that.

Reply:
My long-term goal is actually to do the opposite: make `↑` the simp-normal form and use a typeclass to instantiate the appropriate simp lemmas like `coe_zero`, `coe_one`, etc. below.